Dockerfile

```diff
@@ -21,7 +21,7 @@ WORKDIR /usr/src/app
 
 COPY --chown=node:node package*.json ./
 
-RUN npm ci --only=production
+RUN npm ci --only=production --unsafe-perm
```

Comment:
COPY --chown=node:node package*.json ./ 
-> we think that the permissions problem is here, you need to put permmisions on the package*.json file, check it, if doesn't work i'll approve